ci-operator/step-registry/openshift/hac-dev/e2e/openshift-hac-dev-e2e-commands.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
```

Comment:
What about using
```
set -a
<defining vars>
set +a
```
instead of namely exporting all the variables?

Reply:
Updated, thanks 🙂 that looks clean.

Reply:
unfortunately variable which are used externally is giving error
`In .//ci-operator/step-registry/openshift/hac-dev/e2e/openshift-hac-dev-e2e-commands.sh line 21:
QONTRACT_PASSWORD=$(cat /usr/local/ci-secrets/github/QONTRACT_PASSWORD)
^---------------^ SC2034: QONTRACT_PASSWORD appears unused. Verify use (or export if used externally).` 
 either to go with set -a and namely export those external variables or namely export all the variables.